step-framework-timeseries/src/test/java/step/core/timeseries/TimeSeriesIngestionTest.java

```diff
@@ -168,7 +168,7 @@ public void ingestionWithPartialTTLMatchingTest() {
     @Test
     public void ingestionWithManyBucketsTest() {
         TimeSeries timeSeries = getTimeSeriesWithResolutions(1000, 5000, 30_000);
-        long min = 999999;
+        long min = 0;
```

Comment:
@lucianbaciu I propose to set it to max long, now you changed it to 0. Unless I miss something the evaluation of "min = Math.min(min, b.getMin());" in the for loop will always be 0.

Reply:
@david-stephan auto-merge changed it by some reason. I put it back per your suggestion